### DISCUSSION TITLE
Osu!Lazer input

### DISCUSSION THREAD

**Discussion by Brodester135** [ppy/osu#14671] [Q&A]
So I'm currently a top 600k player on standard Osu! Who loves the game, so I tried to play Osu!Lazer but it seems as if the mouse feels like it drags. For more explanation, whenever I move my mouse, it feels like the cursor gets there a second later, which is fine with low star maps, but I don't play those. It doesn't feel like standard Osu! Do any of you know how to fix?

**Comment by peppy** [comment]
Run the game full screen, and ensure you are running in exclusive mode. Note that this may not be possible right now (if you were using "compatibility mode" on stable, for instance).
